source/Octopus.Client/OctopusAsyncRepository.cs

```diff
@@ -9,12 +9,17 @@
 
 namespace Octopus.Client
 {
-    public static class OctopusRepositoryExtensions
+    public static partial class OctopusRepositoryExtensions
     {
         public static IOctopusAsyncRepository CreateRepository(this IOctopusAsyncClient client, RepositoryScope scope = null)
         {
             return new OctopusAsyncRepository(client, scope);
         }
+
+        public static IOctopusAsyncRepository ForSpace(this IOctopusAsyncRepository repo, SpaceResource space)
```

Comment:
This should return the `IOctopusSpaceAsyncRepository`. So the implementation can just be `return repo.Client.ForSpace(space);`

Reply:
We should also add the analogue of this method for system repositories

```
public static IOctopusSystemAsyncRepository ForSystem(this IOctopusAsyncRepository repo)
{
    return repo.Client.ForSystem();
}
```

Reply:
fixed